components/lock.rst

```diff
@@ -81,57 +81,57 @@ continue the job in another process using the same lock::
     use Symfony\Component\Lock\Lock;
 
     $key = new Key('article.'.$article->getId());
-    $lock = new Lock($key, $this->store, 300, false);
+    $lock = new Lock($key, $this->store, autoRelease: false);
```

Comment:
I did it as I found it easier to understand. I suppose people are using them anyway, so why not use them in the docs. I can revert if you think it would send a signal that they are part of the BC promise.